### PR TITLE
Jael public API

### PR DESCRIPTION
**Pull request by drbeefsupreme**
This PR adds `/docs/reference/vane-apis/jael.md`. It describes `task`s that can
be `%pass`ed to Jael and what `gift`s may be returned in response.

Edit 2020.12.03 - finally looped back to this! My only outstanding question is that the comment above the `%moon` task says its for changing private keys, but the code looks like it changes public keys. I ignored the comment, but I also don't know the details on how moon keys are stored, so perhaps it really does store private keys but uses the `+public-keys:feel` arm.

I also finally came to the conclusion that including `%pass`es that a `task` can trigger might be considered an implementation detail and thus doesn't necessarily belong in an API doc. I'm not really sure on this. I have chosen the worst path, the middle one, and so have a few tasks that say what `%slip`s and `%pass`es they might trigger and some that do not. I would appreciate another perspective on this.

### CODE REVIEW

**Comment by syzygyzer** [comment]
This is ready to be reviewed except for the `%listen` `task`, which somehow escape my list on things to ask about today.

File: reference/vane-apis/jael.md
Comment (baudtack) [inline reference/vane-apis/jael.md:76]: Is what a %slip is explained somewhere else? not saying this is the right place for it but I have no idea what a %slip is. I infer it's something akin to a move?

Comment (drbeefsupreme) [reply reference/vane-apis/jael.md:76]: Yeah it's explained at `tutorials/arvo/arvo.md`. You can find it searching for `%slip` in the search bar, so I think it is fine not linking it here.

File: reference/vane-apis/jael.md
Comment (baudtack) [inline reference/vane-apis/jael.md:240]: What are the private keys that jael holds? are these your networking keys?

**Review by baudtack — APPROVED** [review]
Looks pretty good! Learned more about jael reading this than I knew before so that's nice. Approved pending technical review by a jael expert.

**Review by philipcmonk — APPROVED** [review]
Sorry for not getting around to this... It looks good!

**Pull request merged**
